packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.js

```diff
@@ -27,16 +65,19 @@ class Checkbox extends React.Component {
   };
 
   render() {
-    const { className, onChange, isValid, isDisabled, ...props } = this.props;
+    const { className, onChange, isValid, isDisabled, label, ...props } = this.props;
     return (
-      <input
-        {...props}
-        className={css(styles.check, className)}
-        type="checkbox"
-        onChange={this.handleChange}
-        aria-invalid={!isValid}
-        disabled={isDisabled}
-      />
+      <div className={css(styles.check, className)}>
+        <input
+          {...props}
+          className={css(styles.checkInput)}
+          type="checkbox"
+          onChange={this.handleChange}
+          aria-invalid={!isValid}
+          disabled={isDisabled}
+        />
+        {label && renderLabel(label, props.id, isDisabled)}
```

Comment:
If we get rid of the function (i think we should). Inlining the label down here would be easier to read and follow.

Reply:
I think it (with all of the conditions) would be too complex to put into JSX.

Reply:
I agree with getting rid of the function here to simplify this